### ISSUE TITLE
Raspberry Pi2

### ISSUE THREAD

**Issue by ciprogrammer** [jantimon/meteor-gpio#1]
Package wont install on RPi2

**Comment by retani** [comment]
That's right! I installed meteor on my RPi2 1GB according to this blog post: http://meteor-universal.tumblr.com/post/118809833179/1st-time-installation-meteor-universal-raspbian-wheezy
Meteor works fine. But:

$ meteor add jantimon:gpio
 => Errors while adding packages:             

While checking for jantimon:gpio@0.1.0:
error: No compatible binary build found for this package. Contact the package
author and ask them to publish it for your platform.

**Comment by jantimon** [comment]
hi thanks for the feedback - I have to grab a raspberry to fix this but I am on vacation right now ;)
could you try to use the source code instead of the binary package?

**Comment by retani** [comment]
hi thanks a lot for the quick reply from vacation! Can you give us a hint on how to do that? I have some trouble to understand why there is a binary involved at all - all I see is two javascript files (client/index.js + server/index.js). Isn't it all javascript? I'll try copying the files manually and to install the dependencies.

**Comment by jantimon** [comment]
you are exactly right there are no binaries but meteor asks to pre build packages anyway
install it just like it was a private package

**Comment by retani** [comment]
I took me quite a while to figure it out, but this is how to do it:

install https://github.com/DispatchMe/mgp

```
sudo npm install -g mgp
```

edit git-packages.json

``` json
{
 "jantimon:gpio": {
  "git": "https://github.com/jantimon/meteor-gpio.git"
 }
}
```

run mgp

```
mgp
```

add the private package

```
meteor add jantimon:gpio
```

Thanks!

**Comment by KyleKing** [comment]
Awesome package and @retani thanks for your guide!

**Comment by jantimon** [comment]
How could we add the raspberry package to atmospherejs?
